snomed/com.b2international.snowowl.snomed.exporter.server/src/com/b2international/snowowl/snomed/exporter/server/dsv/SnomedSimpleTypeRefSetDSVExporter.java

```diff
@@ -350,16 +252,16 @@ public File executeDSVExport(OMMonitor monitor) throws SnowowlServiceException {
 					for (String relationshipId : groupedRelationships.get(groupId).keySet()) {
 						stringBuffer.append(delimiter);
 						TreeSet<String> relationships = Sets.newTreeSet();
-						relationships.addAll(executeRelationshipOrDatatypeQuery(QueryType.RELATIONSHIP, conceptCDOId, relationshipId, true, groupId));
-						stringBuffer
-								.append(joinResultsWithDelimiters(relationships, groupedRelationships.get(groupId).get(relationshipId), delimiter, relationshipTargetIdExpected));
+						relationships.addAll(getRelationshipTokens(referencedComponent, relationshipId, groupId));
+						stringBuffer.append(joinResultsWithDelimiters(relationships, groupedRelationships.get(groupId).get(relationshipId), delimiter, includeRelationshipId));
 					}
 				}
 				stringBuffer.append(System.getProperty("line.separator"));
 				os.writeBytes(stringBuffer.toString());
 				remainderMonitor.worked(1);
 			}
-			connection.close();
+			File  zipFile = FileUtils.createZipArchive(file.getParentFile(), Files.createTempFile("export", ".zip").toFile());
```

Comment:
yeah... i know :D 
but this one is clean up by SnomedRefSetDSVExportServerIndication.java#L134
after being sent :)